src/api/app/views/webui2/webui/cloud/configurations/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,6 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+%li.breadcrumb-item.active{ 'aria-current' => 'page' }
```

Comment:
If I'm not wrong, we don't need this `active` here, only in line 5.

Reply:
done